openshift/github-mirror.yaml

```diff
@@ -41,6 +41,11 @@ objects:
                 secretKeyRef:
                     key: db.endpoint
                     name: ${EC_SECRET_NAME}
+            - name: REPLICA_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                    key: db.endpoint
+                    name: ${EC_SECRET_NAME}
```

Comment:
This will be commented for now, right?

Reply:
well no, it is still reading the `db.endpoint`. Which is fine since it is the same as the primary endpoint. Once we set up a `db.replica` I'll add that in and comment it out. So basically you either select `db.endpoint` or `db.replica`, and it's only one line that is commented it out. I didn't put it in yet because I didn't want to reference something that didn't exist yet even in a comment.

Reply:
Right right, didn't notice the secret key name here :)